### PR TITLE
remove haml-coffee dependencies

### PR DESCRIPTION
**Pull request by elinskytap**
https://jira.corp.skytap.com/browse/ENG-64968

haml-coffee has a dependency on optimist caused the vulnerability 
https://github.com/netzpirat/haml-coffee/blob/master/package.json#L39

### CODE REVIEW

**Comment by elinskytap** [comment]
Thanks for sharing the doc! I found there are a few issues I need to fix.

File: package.json [hunk 1]
```diff
@@ -41,36 +41,35 @@
   },
   "homepage": "https://github.com/skytap/minorjs",
   "dependencies": {
```
Comment (elinskytap) [inline package.json]: changes: 
- removing haml-coffee, haml-coffee-loader, express-partials
- adding pug dependency here 
- ordering packages by alphabetic order (altered by yarning)

File: examples/helloworld/lib/templates/frames/index.pug [hunk 2]
```diff
@@ -0,0 +1,17 @@
+//-
+ Copyright 2014 Skytap Inc.
```
Comment (mcostelloskytap) [inline examples/helloworld/lib/templates/frames/index.pug]: It looks like the copyright date changed her. was that intentional?

Comment (elinskytap) [reply examples/helloworld/lib/templates/frames/index.pug]: It was not intentional. I changed the date back!

File: examples/helloworld/lib/templates/hello/index.pug [hunk 3]
```diff
@@ -0,0 +1,17 @@
+//-
+ Copyright 2016 Skytap Inc.
```
Comment (mnightwardskytap) [inline examples/helloworld/lib/templates/hello/index.pug]: Looks like this one got changed to 2016 instead of frames/index.pug?

Comment (elinskytap) [reply examples/helloworld/lib/templates/hello/index.pug]: ahh thanks for catching that

**Pull request merged**